mpdev/internal/apply/deployment_manager.go

```diff
@@ -30,15 +30,37 @@ import (
 // given an autogen.yaml file.
 type DeploymentManagerAutogenTemplate struct {
 	BaseResource
-	AutogenSpec interface{}
-	PartnerID   string `json:"partnerId"`
-	SolutionID  string `json:"solutionId"`
+	Spec spec
 
 	outDir string
 }
 
+type spec struct {
+	DeploymentSpec map[string]interface{} `yaml:"deploymentSpec"`
```

Comment:
should these annotations be `yaml` or `json`? I think `json` works for the YAML parser as well, right?